File: Application/Services/Operations/Finances/Dtos/CardDto.cs
using System;
using Application.Services.Operations.Finances.Dtos.Enums;

namespace Application.Services.Operations.Finances.Dtos
{
    public class CardDto
    {
        public int Id { get; set; }
        public string Holder { get; set; }
        public string Flag { get; set; }
        public decimal Limit { get; set; }
        public TypeCardEnumDto Type { get; set; }
        public string Number { get; set; }
        public int CVC { get; set; }
        public string Description { get; set; }
        public DateTime Validate { get; set; }
        public int BankAccountId { get; set; }
        public BankAccountDto BankAccount { get; set; }

    }
}

File: Views.old/src/components/financial/components/bank-account-cards/dto/card-dto.ts
import { RootBase } from "src/shared/entities-dtos/root-base";
import { CreditCardExpenseInvoiceDto } from "../../credit-card-fixed-expenses/dto/credit-card-expense-invoice-dto";
import { BankAccountDto } from "./bank-account-dto";
import { CreditCardLimitOperationDto } from "./credit-card-limit-operation-dto";
import { TypeCardDtoEnum } from "./enums/type-card-dto.enum";

export class CardDto extends RootBase{
  holder: string;
  flag:string;
  creditLimit: number;
  creditCardLimitOperation:CreditCardLimitOperationDto;
  type: TypeCardDtoEnum;
  number: string;
  cvc: number;
  description:string;
  validate: Date;
  closingDate:Date;
  expiresDate:Date;
  bankAccountId:number;
  bankAccount:BankAccountDto;
  creditCardExpensesInvoices:CreditCardExpenseInvoiceDto[]
}
